src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -28,20 +29,32 @@ const CaptionRow = styled(Row)<{ gap: number; shrink?: number }>`
   height: 100%;
 `
 
+interface CaptionTooltip {
+  content: ReactNode
+  placement?: Placement
```

Comment:
Do you need to pass in placement? Will it not always be bottom, as part of the Caption?
I would rather this interface be removed, and CaptionProps just have `tooltip?: ReactNode`.

Reply:
yes, it's not always bottom. new design has it to the right/left